tests/RazorLight.Tests/RazorLight.Tests.csproj

```diff
@@ -43,7 +34,7 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
```

Comment:
Sorry I didn't notice this earlier - Is there a reason this is 16.2.0 and the other one is 16.4.0?

I recommend, in Visual Studio, right clicking on Solution, Going to Manage Nuget Packages, and seeing if there is any Consolidate recommendations.

Otherwise, this looks like it's close to ready - thanks for all your help.

Reply:
Forget my comment about using Consolidate.  NEver tried it before on a project like this.  Wow, the experience sucks.

Reply:
I see.  16.4 supports .netcoreapp2.1.  That sucks.  OK - looks good to me.